bats/fb-content-katello.bats

```diff
@@ -124,8 +124,22 @@ EOF
   subscription-manager repos --enable="Default_Organization_Test_Product_Zoo" | grep -q "is enabled for this system"
 }
 
-@test "install package from content view" {
-  tPackageInstall walrus && tPackageExists walrus
+@test "install katello-agent" {
+  tPackageInstall katello-agent && tPackageExists katello-agent
+}
+
+@test "start katello-agent" {
+  service goferd start
```

Comment:
I believe this *should* be started automatically (https://github.com/Katello/katello-agent/blob/master/katello-agent.spec#L52) -- so maybe we should do a check that it is running?